src/widgets/_toolbars.scss

```diff
@@ -2,3 +2,15 @@ actionbar,
 .search-bar {
     background-color: bg-color(3);
 }
+
+toolbar separator {
+    background: none;
+    margin: 0 rem(18px);
+}
+
+toolbar paned separator,
+.toolbar .pane-separator {
```

Comment:
where is this selector used? This seems oddly specific

Reply:
Some apps use this to make "dividers" in their toolbars.